packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWCircleButton/CWCircleButton.scss

```diff
@@ -0,0 +1,100 @@
+@import '../../../../../../styles/shared.scss';
+
+.circle-btn-border {
+  width: fit-content;
+  border-radius: 50%;
+
+  &.primary {
+    border: 2px solid $transparent;
+  }
+
+  &.secondary {
+    border: 1px solid $transparent;
+  }
+
+  &.primary,
+  &.secondary {
+    padding: 2px;
+
+    &:focus-within {
+      border: 2px solid $neutral-800;
+
+      .CircleButton.secondary {
+        border: none;
+      }
+    }
+  }
+
+
+
+  .CircleButton {
+    line-height: 0;
+    width: fit-content;
```

Comment:
I think the buttons are smaller versus designs. They are 28px whereas on the design they have 32px 
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/577166d5-d95f-4f62-a4ac-3f43491c26a9)


![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/09d94496-26f9-4d7a-ba06-f825a1bd7a9d)

Reply:
@masvelio 
Nice catch! Fixed!